courses/tasks.py

```diff
@@ -117,3 +121,35 @@ def semester_sync(query='', semester=None):
     load_courses(query=query, semester=semester)
     load_requirements(school='SEAS', semester=semester)
     load_requirements(school='WH', semester=semester)
+
+
+def load_course_status_history(full_path):
```

Comment:
I think it makes more sense to move this function directly into the `management` comment -- the `tasks.py` file is generally for Celery tasks. Don't think we've talked about Celery before, but basically it's a way to spin recurring tasks off into a queue to be worked through one-by-one, which this function doesn't really fit.